client/pages/projects/index.jsx

```diff
@@ -14,62 +14,276 @@ import {
   Col,
   Container,
   Input,
-  Row
+  Row,
+  Alert
 } from "reactstrap";
 
 import "../../public/styles/projects.scss";
+import "../../../api/public/schema/projectSchema";
 
-const DESCRIPTION_LENGTH = 200;
+import Fuse from "fuse.js";
+
+const DESCRIPTION_LENGTH = 150;
 
 export default function ProjectsPage() {
+  const [allProjects, setAllProjects] = useState(null);
   const [projects, setProjects] = useState(null);
+  const [visAlert, setAlert] = useState(false);
+  const [selectedUNGoals, setSelectedUNGoals] = useState(null);
+  const [selectedCountry, setSelectedCountry] = useState(null);
+  const [selectedGrpSize, setSelectedGrpSize] = useState(null);
+  const [selectedDifficulty, setSelectedDifficulty] = useState(null);
+
+  const [userInput, setUserInput] = useState("");
+
+  const handleUNGoals = selectedUNGoals => {
+    setSelectedUNGoals(selectedUNGoals);
+  };
+
+  const handleCountry = selectedCountry => {
+    setSelectedCountry(selectedCountry);
+  };
+
+  const handleGrpSize = selectedGrpSize => {
+    setSelectedGrpSize(selectedGrpSize);
+  };
+
+  const handleDifficulty = selectedDifficulty => {
+    setSelectedDifficulty(selectedDifficulty);
+  };
```

Comment:
these four handlers can be replaced with their corresponding state setters, as they just wrap the state setters without modifiying or accessing properties of the parameter

contrast these with `handleSearchChange` which passes in `userInput.target.value` in instead